pkgs/build-support/compress-drv/default.nix

```diff
@@ -61,18 +71,22 @@ let
     ext: prog:
     assert validProg ext prog;
     ''
-      find -L $out -type f -regextype posix-extended -iregex '.*\.(${formatsPipe})' -print0 \
+      find -L $out -type f -regextype posix-extended -iregex '.*\.(${formatsPipe})' ${extraFindOperands} -print0 \
         | xargs -0 -P$NIX_BUILD_CORES -I{} ${prog}
     '';
-  formatsPipe = builtins.concatStringsSep "|" formats;
+  formatsPipe = lib.concatStringsSep "|" formats;
```

Comment:
curious, why use lib over builtin?

Reply:
In this case it makes really no difference but some lib functions contain backwards compatibility code like https://github.com/NixOS/nixpkgs/blob/c3ede4ac04162b29c0fecc2208c9ceac44eec32e/lib/deprecated/misc.nix#L211-L213 and it is easier to just use lib for everything.